concent_api/core/models.py

```diff
@@ -494,6 +494,70 @@ def clean(self) -> None:
             raise ValidationError({
                 'computation_deadline': "TaskToCompute deadline mismatch"
             })
+        # Validation for every nested message which is stored in Control database
+        # Every nested message must be the same as message stored separately.
+        if deserialized_report_computed_task.task_to_compute != deserialized_task_to_compute:
+            raise ValidationError({
+                'report_computed_task': 'Nested ReportComputedTask with TaskToCompute message '
```

Comment:
According to our coding style, this should be broken differently. Same below.

Reply:
Done